.github/workflows/typescript.yml

```diff
@@ -0,0 +1,37 @@
+name: Typescript
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - 'typescript/apps/*'
+      - 'typescript/packages/*'
+  pull_request:
+    branches:
+      - main
+
+permissions:
+  contents: read
+
+concurrency:
+  group: washboard-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  CI: true
```

Comment:
This is already set by the platform usually (looks like [GH does it too](https://github.blog/changelog/2020-04-15-github-actions-sets-the-ci-environment-variable-to-true/))